src/store/lots/types.ts

```diff
@@ -12,6 +14,16 @@ export interface GetLotsActionInterface extends Action<LotsActionsType> {
   payload: any;
 }
 
+export interface CreateLotsActionInterface extends Action<LotsActionsType> {
+  type: LotsActionsType.CREATE_LOT;
+  payload: any;
```

Comment:
please replace any with corresponding type (around all document)

Reply:
I'll resolve it at next develop iteration